readme.md

```diff
@@ -62,6 +62,13 @@ Default: `true`
 
 Apply bit depth reduction.
 
+##### stripMetadata
+
+Type: `boolean`\
+Default: `true`
+
+Strip Metadata.
```

Comment:
This is unrelated and should be done in a separate pull request. The text should also not be title-case.